src/components/multi-select/__snapshots__/multi-select.test.jsx.snap

```diff
@@ -0,0 +1,77 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<MultiSelect> has defaults 1`] = `
+<body>
+  <div>
+    <div
+      class="container"
+      id="test-id"
+    >
+      <div
+        role="presentation"
+      >
+        <label
+          class="label"
+          for="test-id-autocomplete"
+          id="test-id-label"
+        >
+          test label
+        </label>
+        <div
+          class="control-container"
+        >
+          <div
+            class="form-control-children-container"
+            role="presentation"
+          >
+            <div
+              aria-labelledby="test-id-label"
+              class="tag-list"
+              role="grid"
+            >
+              <input
+                aria-autocomplete="list"
+                aria-controls="test-id-listbox"
+                aria-describedby="test-id-empty"
+                aria-expanded="false"
+                aria-haspopup="listbox"
+                aria-labelledby="test-id-label"
+                autocomplete="off"
+                class="input"
+                id="test-id-autocomplete"
+                role="combobox"
+                value=""
+              />
+            </div>
+            <div
+              class="icon-container"
+            >
+              <svg
+                height="6"
+                role="img"
+                width="10"
+              >
+                <title>
+                  Open multi select options
```

Comment:
It would be interesting to interpolate the field label in here. I think it's a better experience to read:

> Open test label select options

And, if we ensure our accessible items are _unique_ then it means we can `userEvent.click(screen.findByTitle(<unique label>))` instead of having to do other shenanigans like a `getAllByTitle` and indexing the array or finding the field container and then querying the inside.

Reply:
And the test failure message would be nicer to read! Instead of a generic message, it might be faster for developers to read that a particular SVG was not found. 🤔

Reply:
I can make that change!